rapids-cmake/cpm/nvcomp.cmake

```diff
@@ -0,0 +1,141 @@
+#=============================================================================
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+include_guard(GLOBAL)
+
+#[=======================================================================[.rst:
+rapids_cpm_nvcomp
+-----------------
+
+.. versionadded:: v22.06.00
+
+Allow projects to find or build `nvComp` via `CPM` with built-in
+tracking of these dependencies for correct export support.
+
+Uses the version of nvComp :ref:`specified in the version file <cpm_versions>` for consistency
+across all RAPIDS projects.
+
+.. code-block:: cmake
+
+  rapids_cpm_nvcomp( [BUILD_EXPORT_SET <export-name>]
```

Comment:
Do we need to document that all unparsed arguments are forwarded to `rapids_cpm_find`, or is that something that should be obvious?

Reply:
That is the existing behavior across all the `rapids_cpm_<pkg>` commands.

I think it is worthwhile to do that as a separate PR.
I have added an issue for this https://github.com/rapidsai/rapids-cmake/issues/191